src/tracks/gosling-track/gosling-track.ts

```diff
@@ -326,7 +344,7 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
         }
 
         override updateTile(/* tile: Tile */) {} // Never mind about this function for the simplicity.
-        override renderTile(/* tile: Tile */) {} // Never mind about this function for the simplicity.
+        renderTile(/* tile: Tile */) {} // Never mind about this function for the simplicity.
```

Comment:
It is satisfying to see multiple `override` removed ☺️

Reply:
Definitely!